### DISCUSSION TITLE
Information about availability of a fix / enhancement

### DISCUSSION THREAD

**Discussion by RaKuNbg** [microsoft/ALAppExtensions#10348] [Ideas] [answered]
When an issue is accepted and closed in this repository I always get an information saying
"... Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. **We will update this issue with information about availability**..."

But I never got an update with information about availability yet.
It would be great, if the build version could be added to the issue when the fix is released (and I would get a mail) so I don't have to check manually on each release, if a fix is already available (e.g. issue https://github.com/microsoft/ALAppExtensions/issues/8409 that was closed at the beginning of september and is still not included) ==> this would also help other people looking for an already released fix to find out, in what version it had been released.

**Comment by JesperSchulz** [comment, marked as answer]
The message you receive is posted automatically, whenever we close the linked master work item internally as "resolved". When we resolve a master work item, the fix will be released with the next major (coming out of our master branch). Since this issue was resolved in September, the fix will get released with the 2021 Wave 1 release in April.
In cases where we backport the fix to the current version (e.g. 2020 Wave 2 aka version 17.x), the issue will received further updates, where we do specify the version in which the fix will get released. Here's an example where that happened: https://github.com/microsoft/ALAppExtensions/issues/10149.
For this to work, our developers must follow a strict process. In case of events, we need to bundle several master work items into one, as we've got too many to backport them individually. By bundeling them, we loose the links and we cannot post updates to GitHub any more. In this case however, it wasn't an event - the fix simply wasn't backported.

I of course understand that this isn't ideal and we'll look into if we can improve the reporting on progress here on GitHub when time allows.
Thanks for the input!

**Reply by RaKuNbg** [reply]
Hi @JesperSchulz 
thanks for the feedback. That makes things a little bit clearer ;-)
Just to verify that I got you right:
The rule is:
* The fix will be released in the next major release.
* Only if it is a severe problem a fix will be released in a minor update.
Is that correct?

That would mean that we have to wait up to half a year until we get a publisher that we need in a customer project for an extension?!
In my case (also as this one isn't absolutely necessary but makes life easier) even longer (September 2020 to April 2021 = 4 months)

**Reply by JesperSchulz** [reply]
It depends on the type of reported issue. Events will get backported to next minor, while generic extensibility requests usually aren't. You can read more about the expected release date for the individual issue types in our FAQ: https://github.com/Microsoft/ALAppExtensions/blob/master/FAQ.md#when-are-my-reported-issues-going-be-released
In your case, you were unlucky that you just missed the 2020 Wave 2 release and hence had to wait for half a year. If you are blocked by any issue that was closed and not backported, please create a followup issue where you request us to backport. We will of course try to assist you the best we can.
